docs/snippets/react/list-story-with-unchecked-children.ts.mdx

```diff
@@ -0,0 +1,10 @@
+// List.stories.tsx
+
+import React from 'react';
+import { List } from './List';
+import { Unchecked } from './ListItem.stories';
+
+export const OneItem = Template.bind({});
```

Comment:
Template is undefined

Reply:
@DaleSeo if we're going for clarity. Add the `ListProps` to the imports as well.  Turning this:
```js
import { List } from './List';
```

into:

```js
import { List, ListProps } from './List';
```

Import aside we're good to go. Let me know when you addressed the changes and we'll get it merged.

Stay safe

Reply:
@jonniebigodes Good catch! I addressed it.